pyproject.toml

```
@@ -154,3 +154,7 @@ module = [
   "securesystemslib.*",
 ]
 ignore_missing_imports = "True"
+
+[tool.pydocstyle]
+inherit = false
+ignore = "D400,D209,D415,D213,D205,D202,D107,D407,D413,D212,D104,D406,D105,D411,D401,D200,D203"
```

https://github.com/theupdateframework/python-tuf/pull/2281 has been merged. This should work now:
```suggestion
ignore = "D400,D415,D213,D205,D202,D107,D407,D413,D212,D104,D406,D105,D411,D401,D200,D203"
```

---

Please update and we can merge.